src/tiled/invertYCoordinateHelper.h

```diff
@@ -0,0 +1,19 @@
+#ifndef INVERTYCOORDINATEHELPER_H
+#define INVERTYCOORDINATEHELPER_H
+
+#include "preferences.h"                // To get the status of the invertYCoordinate checkBox
+#include "mapdocument.h"                // To access the map
+#include "mapdocumentactionhandler.h"
+
+
+class InvertYCoordinateHelper
+{
+public:
+    float getGridY( float y ) const;
```

Comment:
Coding style: please don't put spaces on the insides of parentheses.

Naming style: please don't prefix methods with "get". I would say these functions should be called `tileY` and `pixelY`. "tile" instead of "grid" since there is no other place in the code where tile coordinates are called "grid" coordinates.

Also please use `qreal` intead of `float`. Using `qreal` matches the Qt API, which is effectively `double`.